src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/stores/ListStore.js

```diff
@@ -526,6 +526,66 @@ export default class ListStore {
         });
     };
 
+    updateSelectionIds(newSelectionIds: Array<string | number>) {
+        const oldSelectionIds = this.selectionIds;
+
+        if (oldSelectionIds === newSelectionIds) {
+            return;
+        }
+
+        const removedValues = oldSelectionIds.filter((x) => !newSelectionIds.includes(x));
+        const addedValues = newSelectionIds.filter((x) => !oldSelectionIds.includes(x));
+        const notLoadedIds = [];
+
+        removedValues.forEach((oldId: string | number) => {
+            this.deselectById(oldId);
+        });
+
+        addedValues.forEach((newId: string | number) => {
+            const row = this.findById(newId);
+
+            if (!row) {
+                notLoadedIds.push(newId);
+
+                return;
+            }
+
+            this.select(row);
+        });
+
+        if (!notLoadedIds.length) {
+            return;
+        }
+
+        const observableOptions = {};
+
+        for (const key in this.observableOptions) {
+            observableOptions[key] = this.observableOptions[key].get();
+        }
+
+        this.setDataLoading(true);
+        const options = {...observableOptions, ...this.options};
+
+        options.selectedIds = newSelectionIds.join(',');
+
+        this.loadingStrategy.load(
+            this.resourceKey,
+            options
```

Comment:
I think instead of all the above code you could simply use `{...this.observableOptions, ...this.options}` here instead, as [it is already done in the `ListStore`](https://github.com/sulu/sulu/blob/0d416ba/src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/stores/ListStore.js#L545).

And I guess you also want to include the `fields` here, so that the correct columns are loaded, so the option would be something like the following:

```javascript
{...this.observableOptions, ...this.options, fields: this.field}
```